WalletWasabi.Gui/Tabs/WalletManager/HardwareWallets/ConnectHardwareWalletViewModel.cs

```diff
@@ -323,11 +323,10 @@ public async Task<KeyManager> LoadKeyManagerAsync()
 				}
 				else
 				{
-					Logger.LogInfo("Hardware wallet was not used previously on this computer. Creating a new wallet file.");
-
-					var prefix = selectedWallet.HardwareWalletInfo is null ? "HardwareWallet" : selectedWallet.HardwareWalletInfo.Model.ToString();
-
+					var prefix = selectedWallet.HardwareWalletInfo?.Model.FriendlyName() ?? HardwareWalletModels.Unknown.FriendlyName();
 					walletName = WalletManager.WalletDirectories.GetNextWalletName(prefix);
+					Logger.LogInfo($"Hardware wallet was not used previously on this computer. New wallet was created with name of {walletName}");
```

Comment:
```suggestion
					Logger.LogInfo($"Hardware wallet was not used previously on this computer. New wallet '{walletName}' was created.");
```

Consider this modification. Apostrophes are good to avoid scenario that the variable contains some leading/trailing whitespace. That's hard to spot it in logs.

Reply:
Done.

Reply:
You can just commit that suggestion via Github here. It's just it may be simpler for you.